src/test/java/io/ebean/dbmigration/ddlgeneration/platform/PlatformDdl_AlterColumnTest.java

```diff
@@ -68,7 +68,7 @@ public void testAlterColumnType() throws Exception {
     assertEquals("alter table mytab alter column acol type varchar(20)", sql);
 
     sql = oraDdl.alterColumnType("mytab", "acol", "varchar(20)");
-    assertEquals("alter table mytab modify acol varchar(20)", sql);
+    assertEquals("alter table mytab modify acol varchar2(20)", sql);
```

Comment:
rob, please take a look at this change - my change did break this test - although I think varchar2 is correct here.